maistra/bazelrc

```diff
@@ -22,6 +22,7 @@ build:aarch64 --linkopt=-fuse-ld=lld
 build:ci-config --local_ram_resources=12288
 build:ci-config --local_cpu_resources=6
 build:ci-config --jobs=3
+build:ci-config --action_env=CXXFLAGS="-include /usr/include/c++/8/limits"
```

Comment:
this doesn't seem right

Reply:
ok, I will close this PR. let me check in the openshift/release PR and export the CXXFLAGS setting.